app/uk/gov/hmrc/helptosave/actors/UCThresholdManager.scala

```diff
@@ -88,79 +67,31 @@ class UCThresholdManager(thresholdConnectorProxy: ActorRef,
       getValueFromDES() pipeTo self
 
     case GetDESThresholdValueResponse(result) ⇒
-      handleDESThresholdValue(result)
-
-    case GetMongoThresholdValue ⇒
-      logger.info("Trying to get UC threshold value from mongo")
-      getValueFromMongo() pipeTo self
-
-    case GetMongoThresholdValueResponse(result) ⇒
-      handleMongoThresholdValue(result)
+      result.fold(
+        { e ⇒
+          pagerDutyAlerting.alert("Could not obtain initial UC threshold value from DES")
```

Comment:
do we need to do this here ?  or inside of `fold()` below ? because if des is down and if we retry several times with in a minute, then it causes several alerts  (number of alerts actually depend on how frequent the retries are, anyway)

Reply:
Also can you rename `UCThresholdConnectorProxy` to something that ends with ***Actor  due to this class not being a `Proxy` and its placed inside `actor` package ?